pkg/controller/directvolumemigrationprogress/directvolumemigrationprogress_controller.go

```diff
@@ -195,7 +199,7 @@ func (r *RsyncPodProgressTask) Run() error {
 		if err != nil {
 			return liberr.Wrap(err)
 		}
-		rsyncPodStatus, err := r.getRsyncClientContainerStatus(pod)
+		rsyncPodStatus, err := r.getRsyncClientContainerStatus(pod, r)
```

Comment:
@shubham-pampattiwar Injecting the dependency here (duplication of `r`) feels like we need to change the design of struct `RsyncPodProgressTask`. But for now, this looks OK to me.